Sources/ExecutorType.swift

```diff
@@ -55,6 +83,12 @@ struct QueueExecutor: ExecutorType {
 
 }
 
+/// An operation queue manages a number of operation objects, making higher-
```

Comment:
Hilariously, neither. The "NSOperationQueue Extension Reference" does not include this doc at all.

Moved, it is the dumb `higher- level`. I'll fix it.